### ISSUE TITLE
Images related to amplify-ios do not have informative alt text

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#5715]
**Describe the content issue**:
Images listed below don't have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

If the image is for decoration then having an empty string, `![](...)`, would be okay, provided that they are non-essential to understanding the content and purpose of the page.

If the image is being used to display info, then we need to have more descriptive text for screen readers. For example, the file `src/fragments/lib/project-setup/ios/create-application/10_createProject.mdx` has an image on line 7 that has `GSA` as its alt text, which isn't descriptive or helpful for people who use screen readers. Link to the page where this image is seen is found here here, in the step 1 section: https://docs.amplify.aws/lib/project-setup/create-application/q/platform/ios/#project-setup. The text above the image describes clearly how to set up the app, so it's possible we could just make this image decorative by having no alt text. If we wanted to add alt text though, maybe we could have something like: `Screenshot of the create new project wizard with the application type App selected`. 

I was thinking that this issue could be used to track which alt text needs to be updated or can be ignored. Please let me know if you have any questions!

Some resources to help with this:
- https://webaim.org/techniques/alttext/
- https://supercooldesign.co.uk/blog/how-to-write-good-alt-text

```
src/fragments/guides/location-service/setting-up-your-app-ios.mdx:
   95:     ![Amazon Location Service - Copy iOS project files](/images/als/ios-copy-project-files.png)
  105:     ![Amazon Location Service - Adding files options](/images/als/ios-adding-files-options.png)
  196:     ![Amazon Location Service - Create place index](/images/als/create-place-index.png)
  200:     ![Amazon Location Service - Place index](/images/als/my-place-index.png)

src/fragments/guides/location-service/tracking-device-location-ios.mdx:
   9:       ![Amazon Location Service - Create tracker](/images/als/create-tracker.png)
  13:       ![Amazon Location Service - Tracker](/images/als/my-tracker.png)

src/fragments/lib/geo/ios/search/30_location_search_map.mdx:
  51: ![AMLMapView with AMLSearchBar](/images/ios-geocoder-search-box-map.png)
  73: ![AMLMapView featureImage view modifier](/images/ios-geocoder-custom-images.png)

src/fragments/lib/project-setup/ios/create-application/10_createProject.mdx:
   7: ![GSA](/images/project-setup/20_1_createProject.png)
  15: ![GSA](/images/project-setup/20_2_createProject.png)
  18: ![GSA](/images/project-setup/20_3_createProject.png)

src/fragments/lib/project-setup/ios/create-application/20_install_libraries.mdx:
   3: ![Add package dependency](/images/project-setup/20_4_add-package-dependency.png)
   9: ![Search for repo](/images/project-setup/20_5_search-amplify-repo.png)
  22: ![Select dependencies](/images/project-setup/20_7_select-dependencies.png)

src/fragments/lib/push-notifications/ios/getting_started/20_set_entitlements.mdx:
  22: ![Add capability](/images/push-notifications/xcode-entitlements/01_add-capability.png)
  32: ![Add push](/images/push-notifications/xcode-entitlements/02_add-push.png)
  36: ![App sandbox](/images/push-notifications/xcode-entitlements/05_app-sandbox.png)
  40: ![Outgoing connections](/images/push-notifications/xcode-entitlements/06_app-sandbox-outgoing-connections.png)

src/fragments/lib/push-notifications/ios/getting_started/ios-set-entitlements.mdx:
   3: ![Add capability](/images/push-notifications/xcode-entitlements/01_add-capability.png)
  13: ![Add push](/images/push-notifications/xcode-entitlements/02_add-push.png)
  17: ![Add background](/images/push-notifications/xcode-entitlements/03_add-background.png)
  23: ![Remote notifications](/images/push-notifications/xcode-entitlements/04_background-remote-notifications.png)

src/fragments/lib/push-notifications/ios/remote_media/10_remote_media.mdx:
  13: ![New target](/images/push-notifications/service-extension/01_new-target.png)
  17: ![Target type](/images/push-notifications/service-extension/02_target-type.png)
  21: ![Target name](/images/push-notifications/service-extension/03_target-name.png)
  25: ![Add library](/images/push-notifications/service-extension/04_add-library.png)
  29: ![Notification Service](/images/push-notifications/service-extension/05_notification-service.png)

src/fragments/lib-v1/geo/ios/search/30_location_search_map.mdx:
  48: ![AMLMapView with AMLSearchBar](/images/ios-geocoder-search-box-map.png)
  70: ![AMLMapView featureImage view modifier](/images/ios-geocoder-custom-images.png)

src/fragments/lib-v1/project-setup/ios/combine/combine.mdx:
  102: ![image](/images/combine-xcode.png)

src/fragments/lib-v1/project-setup/ios/create-application/10_createProject.mdx:
   5: ![GSA](/images/project-setup/20_1_createProject.png)
  13: ![GSA](/images/project-setup/20_2_createProject.png)
  16: ![GSA](/images/project-setup/20_3_createProject.png)

src/fragments/lib-v1/project-setup/ios/create-application/20_install_libraries.mdx:
   7: ![Add package dependency](/images/project-setup/20_4_add-package-dependency.png)
  13: ![Search for repo](/images/project-setup/20_5_search-amplify-repo.png)
  26: ![Select dependencies](/images/project-setup/20_7_select-dependencies.png)

src/fragments/lib-v1/project-setup/ios/create-application/31_provisionBackend.mdx:
   3: ![GSA](/images/project-setup/50_1_dragDrop.png)
  11: ![GSA](/images/project-setup/50_2_addFiles.png)

src/fragments/sdk/analytics/ios/getting-started.mdx:
  147:    ![getting-started-analytics](/images/getting-started-analytics.png)

src/fragments/sdk/api/ios/graphql.mdx:
  257: ![Image](/images/appsync-architecture.png)

src/fragments/sdk/auth/ios/federated-identities.mdx:
  213: ![Image](/images/dev-auth-ids-console-settings.png)

src/fragments/sdk/pubsub/ios/getting-started.mdx:
  67: ![Alt text](/images/iot_attach_policy.png)

src/fragments/start/getting-started/ios/add-api.mdx:
  142:     ![](/images/lib/getting-started/ios/set-up-ios-appsync-query.png)
  169:     ![](/images/lib/getting-started/ios/set-up-ios-appsync-create.png)

src/fragments/start/getting-started/ios/setup.mdx:
  25:   ![](/images/lib/getting-started/ios/set-up-ios-select-project-template.png)
  33:   ![](/images/lib/getting-started/ios/set-up-ios-studio-configure-your-project.png)
  54:     ![Add package dependency](/images/lib/getting-started/ios/set-up-ios-add-package-dependency.png)
  60:     ![Add the Amplify package](/images/lib/getting-started/ios/set-up-ios-search-amplify-repo.png)
  64:     ![Select dependencies](/images/lib/getting-started/ios/set-up-ios-select-dependencies.png)
  74: ![Edit dependencies](/images/lib/getting-started/ios/set-up-ios-edit_dependencies.png)
```
